### PR TITLE
atop: add page

### PR DESCRIPTION
**Pull request by einverne**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by yutyo — APPROVED** [review]
LGTM

**Review by kbdharun — APPROVED** [review]

File: pages/linux/atop.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Output background process information:
+
+`atop -c`
+
+- Output thread-specific resource utilization information:
+
+`atop -y`
+
+- Output number of processes for each user:
```
Comment (kbdharun) [inline pages/linux/atop.md:26]:
```suggestion
- Output the number of processes for each user:
```

File: pages/linux/atop.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`atop -au`
+
+- Get help about interactive commands:
```
Comment (kbdharun) [inline pages/linux/atop.md:30]:
```suggestion
- Get help with interactive commands:
```

**Review by kbdharun — COMMENTED** [review]
Just some small suggestions.

File: pages/linux/atop.md (hunk 2 above)
Comment (kbdharun) [reply pages/linux/atop.md:30]: Or display help is fine too.

File: pages/linux/atop.md (hunk 1 above)
Comment (navarroaxel) [reply pages/linux/atop.md:26]:
```suggestion
- Start and display the number of processes for each user:
```

File: pages/linux/atop.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# atop
+
+> Linux system and process monitor.
+> More information: <https://linux.die.net/man/1/atop>.
+
+- Start atop:
```
[the remark this answers is not in the document]
Comment (navarroaxel) [reply pages/linux/atop.md:6]: Just start
```suggestion
- Start:
```

File: pages/linux/atop.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`atop`
+
+- Output memory consumption for each process:
```
Comment (navarroaxel) [inline pages/linux/atop.md:10]:
```suggestion
- Start and display memory consumption for each process:
```

File: pages/linux/atop.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`atop -m`
+
+- Output disk information:
```
Comment (navarroaxel) [inline pages/linux/atop.md:14]:
```suggestion
- Start and display disk information:
```

File: pages/linux/atop.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+`atop -d`
+
+- Output background process information:
```
Comment (navarroaxel) [inline pages/linux/atop.md:18]:
```suggestion
- Start and display background process information:
```

File: pages/linux/atop.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+`atop -c`
+
+- Output thread-specific resource utilization information:
```
Comment (navarroaxel) [inline pages/linux/atop.md:22]:
```suggestion
- Start and display thread-specific resource utilization information:
```

**Review by kbdharun — APPROVED** [review]

File: pages/linux/atop.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
+# atop
+
+> Linux system and process monitor.
+> More information: <https://linux.die.net/man/1/atop>.
```
Comment (kbdharun) [inline pages/linux/atop.md:4]:
```suggestion
> More information: <https://manned.org/atop>.
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**